### ISSUE TITLE
[BUG] DroppingItemEvent isn't called when a player dies with items in their inventory.

### ISSUE THREAD

**Issue by Cyanox62** [Exiled-Team/EXILED#717]
**Describe the bug**
DroppingItemEvent isn't called when a player dies with items in their inventory.
_NOTE: I'm currently working on my first Exiled 3.0 plugin, it's entirely possible I've just forgotten something simple due to lack of experience._

**To Reproduce**
Write a debug line on the event and kill a player, notice it's not executed.

**Expected behavior**
The event is fired upon a player dying with items in their inventory.

**Server logs**
N/A

**EXILED Version ("latest" is not a version):**
3.0.0-alpha.83

**Results of `show plugins` command in console:**
N/A

**Additional context**
N/A

**Comment by sanyae2439** [comment]
confirmed bug.
it's not implemented (because UserCode_CmdDropItem doesnt call when onplayerdied)

**Comment by PintTheDragon** [comment]
I'd say this should be a separate event. The player isn't actually dropping the items.

**Comment by joker-119** [comment]
Agreed, and if for ex you tried to stop 'dropping' the item when they die it either just wouldn't work, or probably break something. Perhaps adding ev.ItemsToDrop to Player.Dying? Then you could specifically remove items that are meant to drop, or replace them etc?

**Comment by NaoUnderscore** [comment]
I guess we should make an event for that, because I think that `DroppingItemEventArgs` refers to a player that drops an item of his own accord.
About your idea Joker, I'd put that property inside the new event, cause your should be able to handle the whole behaviour, otherwise we'd get an event that is being called and an `ev.IsAllowed` property, which is pretty useless since we're handling the whole behaviour, therefore even the items being dropped.

**Comment by PintTheDragon** [comment]
Yeah, just add it to death events. No need to make a new event for it.

**Comment by NaoUnderscore** [comment]
Ye, since a new event would be useless at this point.

**Comment by joker-119** [comment]
I'd favor not making a new event. I don't feel like there's a need for it. There's no other behavior that's really needed I feel, just a list of items that are to be dropped when they die. Unless the dropping of those items

**Comment by NaoUnderscore** [comment]
@Cyanox62 This has been fixed in #757
